install/minikube-installation-guide.md

```diff
@@ -0,0 +1,240 @@
+    * [Check Minikube is running](#check-minikube-is-running)
+    * [Installing Kata Containers](#installing-kata-containers)
+    * [Enabling Kata Containers](#enabling-kata-containers)
+        * [Register the runtime](#register-the-runtime)
+    * [Testing Kata Containers](#testing-kata-containers)
+    * [Wrapping up](#wrapping-up)
+
+## Introduction
+
+[Minikube](https://kubernetes.io/docs/setup/minikube/) is an easy way to try out a Kubernetes (k8s)
+cluster locally. It utilises running a single node Kubernetes stack in a local VM.
+
+[Kata Containers](https://github.com/kata-containers) can be installed into a Minikube cluster using
+[kata-deploy](https://github.com/kata-containers/packaging/tree/master/kata-deploy).
+
+This document details the pre-requisites, installation steps, and how to check
+the installation has been successful.
+
+## Prerequisites
+
+This installation guide has only been verified under a Minikube Linux installation, using the
+[kvm2](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#kvm2-driver) driver.
+
+This installation guide may not work for MacOS installations of Minikube, due to the lack of
+nested virtualization support on that platform.
+
+This installation guide has not been tested on a Windows installation.
+
+See the [Minikube installation guide](https://kubernetes.io/docs/tasks/tools/install-minikube/)
+for details on installation, whilst paying attention to the following [Setup](#setting-up-minikube)
+section of this document. Also pay attention to the following
+[Nested Virtualization](#checking-for-nested-virtualization) section of this document, as
+Kata Containers will not function under Minikube if this is not enabled on your system.
+
+## Setting up Minikube
+
+To enable Kata Containers under Minikube, you need to add a few configuration options to the
+default Minikube setup. This is easily accomplished, as Minikube supports them on the setup commandline.
+
+Here are the features, and why you need them:
+
+| what | why |
+| ---- | --- |
+| `--vm-driver kvm2` | The host VM driver |
+| `--memory 6144` | Allocate sufficient memory, as Kata Containers default to 1 or 2Gb |
+| `--network-plugin=cni` | As recommended for [minikube CRI-o](https://kubernetes.io/docs/setup/minikube/#cri-o) |
+| `--enable-default-cni` | As recommended for [minikube CRI-o](https://kubernetes.io/docs/setup/minikube/#cri-o) |
+| `--container-runtime=cri-o` | Using CRI-O for Kata |
+| `--bootstrapper=kubeadm` | As recommended for [minikube CRI-o](https://kubernetes.io/docs/setup/minikube/#cri-o) |
+
+> **Note:** This example deploys Kata Containers using CRI-O. The same procedure should also work
+> for a `containerd` based Kubernetes installation, but is not covered in this document.
+
+> **Note:** Prior to Minikube/Kubernetes v1.14, the beta `RuntimeClass` feature also needed enabling with
+> the following. Recent Minikube/Kubernetes installations should no longer require this setting.
```

Comment:
Technically, can't you drop this sentence entirely as you've already explained that `RuntimeClass` is only needed for k8s versions older than v1.14?

Reply:
Could we change "Recent" for the actual version details? Recent gets old fast round here ;)